docs_user/modules/openstack-edpm_adoption.adoc

```diff
@@ -108,6 +108,44 @@ spec:
 EOF
 ----
 
+* In case when neutron-sriov-nic-agent is running on the existing compute nodes,
+  physical device mappings needs to be checked and set the same in the
+  OpenStackDataPlaneNodeSet CR
+
+----
+openstack network agent list
```

Comment:
This is great but isn't it too late (assuming the reader goes from start to the end of the guide)? I believe when they get to EDPM adoption, their API services are already down and the command below won't work. Should we instruct them to pull the data earlier, somewhere in Network section perhaps?

Reply:
You are totally right. I checked it once again and just added link to the "Pulling openstack configuration section" for more details. I think this should be enough.

Reply:
I think the newer version is still too late. We stop API services very early in the process, that's why it's important to pull the sriov configuration data prior to any actual adoption steps. At the very least, we shoud inform the reader in the Pulling the configuration section somehow that they have to take care of SR-IOV (either as a link from that earlier section to here, or by moving the text about pulling the sriov configuration there directly; I prefer the latter.)

Reply:
I changed it once again, adding info about fetching sriov agent's configuration in the "pull openstack configuration" section and linking to the edpm adoption. Instead of using neutron API I wrote it in the way that data is fetched directly from the neutron DB which is consistent with how other things are fetched there (like e.g. hostnames of the nova hypervisor names, etc.)